Gemfile

```diff
@@ -2,7 +2,7 @@
 source 'https://rubygems.org'
 
 group :development do
-  gem 'rspec', '~> 3.0'
+  gem 'rspec', '~> 3.5'
```

Comment:
Ah, so I had not intended this to be part of this PR. But this was something I had to do in order to get things running. See #1062